tests/server/endpoints/covidcast_utils/test_trend.py

```diff
@@ -0,0 +1,61 @@
+# standard library
+import unittest
+
+from delphi.epidata.server.endpoints.covidcast_utils.trend import compute_trend_value, compute_trend_class, TrendEnum, compute_trend, Trend
+
+
+class UnitTests(unittest.TestCase):
+    def test_compute_trend_value(self):
```

Comment:
Add a check for when basis == min, basis != current.

& we never expect a case when basis == min, basis > current, right?

Reply:
how can `basis > current and basis == min` be true, it would imply that `min > current`, so that the minimal value is larger than some other value

added the other tests